algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -367,3 +367,9 @@ case class QTree[A](
     (n.toDouble(ll) / luc, n.toDouble(uu) / ulc)
   }
 }
+
+case class QTreeAggregator(percentile: Double, k: Int) extends Aggregator[Double, QTree[Double], (Double, Double)] {
```

Comment:
the tree will have at most `2^k` nodes. So, like with HLL, the reasonable range is somewhere from 7...12 or so. 8 or maybe 9 seems like a decent default to me.